### ISSUE TITLE
[stable/fluentd-elasticsearch] DaemonSet ConfigMap Volume goes beyond 63 char limit

### ISSUE THREAD

**Issue by AmundsenJunior** [helm/charts#7317]
**Is this a request for help?**: Yes

---

**Is this a BUG REPORT or FEATURE REQUEST?** (choose one): Bug Report

**Version of Helm and Kubernetes**: Kubernetes v1.8.9, Helm v2.9.1

**Which chart**: stable/fluentd-elasticsearch

**What happened**: Attempting to `helm install` with a long release name produced an error:

```
message: 'Error: release test-0-1-5-pfsan-251-c48a86e-6-app failed: DaemonSet.apps "test-0-1-5-pfsan-251-c48a86e-6-app-fluentd-elasticsearch" is invalid: [spec.template.spec.volumes[3].name: Invalid value: "config-volume-test-0-1-5-pfsan-251-c48a86e-6-app-fluentd-elasticsearch": must be no more than 63 characters, spec.template.spec.containers[0].volumeMounts[3].name: Not found: "config-volume-test-0-1-5-pfsan-251-c48a86e-6-app-fluentd-elasticsearch"]
```

This is due to the configuration of the DaemonSet ConfigMap Volume name, where it appends "config-volume-" onto the "fluentd-elasticsearch.fullname" template value that gets truncated to 63 characters.

**What you expected to happen**: Should've successfully installed.

**How to reproduce it** (as minimally and precisely as possible): helm install with a release name longer than 63 characters to get the full truncated value for "fluentd-elasticsearch.fullname" template. Helm install should then fail where the DaemonSet ConfigMap Volume name is longer than 63 characters.

**Comment by AmundsenJunior** [comment]
Not stale. PR awaiting approval.

**Comment by AmundsenJunior** [comment]
Not stale. PR awaiting approval.

**Comment by AmundsenJunior** [comment]
Closing, as problem of config volume name length was corrected as part of #8923.
